server/test/unit/server/webservices/views/test_permissions.py

```diff
@@ -95,17 +96,17 @@ def test_grant_to_user(self, mock_factory, mock_resp):
                 new=assert_auth_EXECUTE())
     def test_grant_to_user_invalid_param(self):
         """
-        Test grant permissions to user with invalid params.
+        Test grant permissions to user with missing required params.
         """
         request = mock.MagicMock()
         request.body = json.dumps({'operations': ['READ'], 'resource': '/v2/some/'})
         grant = GrantToUserView()
         try:
             grant.post(request)
-        except InvalidValue, response:
+        except MissingValue, response:
             pass
         else:
-            raise AssertionError("InvalidValue should be raised with invalid params")
+            raise AssertionError("MissingValue should be raised with missing params")
```

Comment:
These `raise AssertionError` statements could be converted to `self.fail()` statements which I think would be slightly better. Similarly throughout the PR.